### PR TITLE
Remove Unused Solution Configurations

### PR DESCRIPTION
**Pull request by Brett208**
- Leave Debug as an option
 - Delete all but Release MinSize
 - Rename Release MinSize to Release

Tested rebuilding code in Release mode and new file was compatible with OP2Mapper.

### CODE REVIEW

**Comment by Brett208** [comment]
I reviewed the differences between Release MinDependency and Release MinSize. 

The only notable setting difference was in the property UseOfAtl (Specifies whether the ATL project will statically or dynamically link to the ATL .dll.) ATL stands for Active Template Library, however I have no experience with it.

- Release MinDependency: UseOfAtl = static
- Release MinSize: UseOfAtl = Dynamic

We can add another commit switching UseOfAtl to static in the new Release Build Configuration if that is more appropriate.

Also, I would recommend a rebase and merge for this pull request.

File: OP2Editor.vcxproj [hunk 1]
```diff
@@ -34,260 +18,36 @@
...
-    <OutDir>.\Debug\</OutDir>
-    <IntDir>.\Debug\</IntDir>
-    <LinkIncremental>true</LinkIncremental>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Unicode Release MinDependency|Win32'">
-    <OutDir>.\ReleaseUMinDependency\</OutDir>
-    <IntDir>.\ReleaseUMinDependency\</IntDir>
-    <LinkIncremental>false</LinkIncremental>
-  </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release MinSize|Win32'">
```
Comment (DanRStevens) [inline OP2Editor.vcxproj]: Just want to verify if "Release MinSize" should be renamed to "Release" here.

**Review by DanRStevens — APPROVED** [review]
Might be useful to search for stray references to old setting names, just to see if anything was missed, or if old cruft is still around that is no longer being used. A tool such as [grepWin](https://tools.stefankueng.com/grepWin.html) may be useful for that.

Overall though, the idea looks sound, and the ultimate test is if the code compiles cleanly with the new project settings, and the result is usable.

It's probably fine if a few settings are changed or dropped. There tends to be different settings available between compiler versions anyway. We can always adjust things at a future date if needed.

If I recall correctly, the project was built without depending on the ATL. Hence that ATL setting should be largely irrelevant. The ATL can take care of a lot of COM boilerplate code. As I was learning COM when I started the project, I avoided extra libraries that hid details of what was going on, and wrote stuff manually.

**Comment by Brett208** [comment]
Sloppy work on my part. I found a second missed MinSize besides the one you saw when rescanning through. I usually use VS2017 for renaming stuff like this, but I don't think it is smart to edit the solution and project files while they are in use so I did it manually.

I'll check out grepWin and verify it still builds before merging.

Thanks.

**Pull request merged**